### PR TITLE
Re-add error handling for initialize

### PR DESCRIPTION
**Pull request by cmfcmf**
This isn't thoroughly tested :(

### CODE REVIEW

File: src/node_lib.h [hunk 1]
```diff
@@ -83,7 +83,7 @@ bool eventLoopIsRunning();
  * @param allow_repl Controls whether the node.js REPL gets spawned when stdin
  * is an interactive terminal.
```
Comment (Johannes-Schneider) [inline src/node_lib.h:84]: `@return` is missing. Please add it.

File: src/node_lib.h [hunk 2]
```diff
@@ -103,7 +103,7 @@ NODE_EXTERN void Initialize(const std::string& program_name = "node_lib",
...
```
Comment (Johannes-Schneider) [inline src/node_lib.h:105]: `@return` is missing. Please add it.

**Review by Johannes-Schneider — CHANGES_REQUESTED** [review]

File: src/node_lib.h (hunk 1 above)
Comment (cmfcmf) [reply src/node_lib.h:84]: fixed in https://github.com/hpicgs/node/pull/80/commits/737c01556baa0b0cde76e98206cd187936a8ed57

File: src/node_lib.h (hunk 2 above)
Comment (cmfcmf) [reply src/node_lib.h:105]: fixed in https://github.com/hpicgs/node/pull/80/commits/737c01556baa0b0cde76e98206cd187936a8ed57

**Review by Johannes-Schneider — APPROVED** [review]

**Pull request merged**